### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by marxriedl**
Readme of the bootstrap repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
-# go-clang-phoenix-bootstrap
+# go-clang-phoenix-bootstrap [![GoDoc](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap?status.png)](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap) [![Build Status](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap.svg?branch=master)](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap) [![Coverage Status](https://coveralls.io/repos/zimmski/go-clang-phoenix-bootstrap-gen/badge.png?branch=master)](https://coveralls.io/r/zimmski/go-clang-phoenix-bootstrap?branch=master)
```
Comment (zimmski) [inline README.md]: Remove the coveralls badge since we do not need it.

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+
+Native Go bindings for the C API of Clang.
```
Comment (zimmski) [inline README.md]: IMHO I find `Native Go bindings for Clang's C API.` better

File: README.md [hunk 3]
```diff
@@ -1 +1,38 @@
...
+
+#Usage
```
Comment (zimmski) [inline README.md]: Space after `#`

File: README.md [hunk 4]
```diff
@@ -1 +1,38 @@
...
+
+There is no direct way to use this repo other than for bootstrapping new go-clang-bindings. 
+Look over to [gen](https://github.com/zimmski/go-clang-phoenix-gen) to discover how to generate a new go-clang-binding.
+
+## Where are the bindings?
```
Comment (zimmski) [inline README.md]: `I need bindings for a different Clang version`

Also, just link to https://github.com/zimmski/go-clang-phoenix-gen#where-are-the-bindings Listing all versions here would mean that we have to recreate each version repository for just one new version.

File: README.md [hunk 5]
```diff
@@ -1 +1,38 @@
-# go-clang-phoenix-bootstrap
+# go-clang-phoenix-bootstrap [![GoDoc](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap?status.png)](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap) [![Build Status](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap.svg?branch=master)](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap) [![Coverage Status](https://coveralls.io/repos/zimmski/go-clang-phoenix-bootstrap-gen/badge.png?branch=master)](https://coveralls.io/r/zimmski/go-clang-phoenix-bootstrap?branch=master)
+
+Native Go bindings for the C API of Clang.
+
+#Usage
+
+There is no direct way to use this repo other than for bootstrapping new go-clang-bindings.
```
Comment (zimmski) [inline README.md]: Please rewrite this section so that we can just copy this README for all version repositories.

File: README.md [hunk 6]
```diff
@@ -1 +1,38 @@
-# go-clang-phoenix-bootstrap
+# go-clang-phoenix-bootstrap [![GoDoc](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap?status.png)](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap) [![Build Status](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap.svg?branch=master)](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap) [![Coverage Status](https://coveralls.io/repos/zimmski/go-clang-phoenix-bootstrap-gen/badge.png?branch=master)](https://coveralls.io/r/zimmski/go-clang-phoenix-bootstrap?branch=master)
+
+Native Go bindings for the C API of Clang.
+
+#Usage
+
+There is no direct way to use this repo other than for bootstrapping new go-clang-bindings. 
+Look over to [gen](https://github.com/zimmski/go-clang-phoenix-gen) to discover how to generate a new go-clang-binding.
```
Comment (zimmski) [inline README.md]: This is mentioned in a different section, just remove this sentence.

File: README.md [hunk 7]
```diff
@@ -1 +1,38 @@
...
+
+## Installation
+
+```bash
+CGO_CFLAGS="-I`llvm-config --includedir`" \
+  CGO_LDFLAGS="-L`llvm-config --libdir`" \
+  go get github.com/zimmski/go-clang-phoenix-bootstrap
+```
+
+## Example
```
Comment (zimmski) [inline README.md]: Maybe `Example usage` is a better heading?

File: README.md [hunk 8]
```diff
@@ -1 +1,38 @@
...
+
+An example on how to use the AST visitor of Clang can be found in [/cmd/go-clang-dump/main.go](/cmd/go-clang-dump/main.go)
+
+## Issue Tracking
```
Comment (zimmski) [inline README.md]: Please use `I found a bug/missing a feature in go-clang` here which is more explicit.

File: README.md [hunk 9]
```diff
@@ -1 +1,38 @@
...
+**DO NOT** use the issue tracker of this respository, use instead the issue tracker of the https://github.com/zimmski/go-clang-phoenix-gen repository.
```
Comment (zimmski) [inline README.md]: The `DO NOT...` sentence is not needed `Use the issue tracker of ..` is sufficient.

File: README.md (hunk 1 above)
Comment (marxriedl) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (marxriedl) [inline README.md]: done

File: README.md [hunk 10]
```diff
@@ -1 +1,38 @@
...
+
+## Where are the bindings?
+
+The Go bindings are placed in their own repositories to provide the correct bindings for the corresponding Clang version:
+
+- [v3.4](https://github.com/zimmski/go-clang-phoenix-v3.4)
+- [v3.6](https://github.com/zimmski/go-clang-phoenix-v3.6)
+- [v3.7](https://github.com/zimmski/go-clang-phoenix-v3.7)
+
+## Where is the generation?
```
Comment (zimmski) [inline README.md]: I think that `How is the binding generated?` is a better heading

File: README.md [hunk 11]
```diff
@@ -1 +1,38 @@
...
+```
+
+## Example
+
+An example on how to use the AST visitor of Clang can be found in [/cmd/go-clang-dump/main.go](/cmd/go-clang-dump/main.go)
+
+## Issue Tracking
+**DO NOT** use the issue tracker of this respository, use instead the issue tracker of the https://github.com/zimmski/go-clang-phoenix-gen repository.
+
+## Warning
```
Comment (zimmski) [inline README.md]: Please redo this section so that it describes the `update process`

File: README.md (hunk 4 above)
Comment (marxriedl) [inline README.md]: done

File: README.md (hunk 6 above)
Comment (marxriedl) [inline README.md]: done

File: README.md (hunk 7 above)
Comment (marxriedl) [inline README.md]: would clash with the other headings

File: README.md (hunk 8 above)
Comment (marxriedl) [inline README.md]: done

File: README.md (hunk 9 above)
Comment (marxriedl) [inline README.md]: done

**Comment by zimmski** [comment]
I think the sections need to be reordered. A normal user does not care how the bindings are generated. The main questions are definitely:
- What is this?
- How do I install it?
- How do I use it?
- How do I report problems and missing features?

**Pull request merged**